cppcache/CMakeLists.txt

```diff
@@ -24,6 +24,28 @@ if (CMAKE_USE_PTHREADS_INIT)
   check_function_exists("pthread_setname_np" HAVE_pthread_setname_np)
 endif()
 
+# Add clang-tidy if available
+if(CMAKE_VERSION VERSION_GREATER 3.6)
```

Comment:
So we used to have something like this in CMake before. It was removed because clang-tidy doesn't handle parallel execution well. It will "fix" the same header included via two different source files in parallel and produce garbage output.

Reply:
I'll remove the "fix" option but leave generating the report in place. It still fails the build if the check doesn't pass.